app/helpers/application_helper/toolbar/ems_storage_center.rb

```diff
@@ -67,4 +73,40 @@ class ApplicationHelper::Toolbar::EmsStorageCenter < ApplicationHelper::Toolbar:
                    ]
                  ),
                ])
+  button_group('ems_storage_view', [
+                 twostate(
+                   :view_dashboard,
+                   'fa fa-tachometer fa-1xplus',
+                   N_('Dashboard View'),
+                   nil,
+                   :url       => "/",
+                   :url_parms => "?display=dashboard",
+                   :klass     => ApplicationHelper::Button::ViewDashboard
+                 ),
+                 twostate(
+                   :view_summary,
+                   'fa fa-th-list',
+                   N_('Summary View'),
+                   nil,
+                   :url       => "/",
+                   :url_parms => "?display=main"
+                 ),
+               ])
+  # button_group('ems_storage_authentication', [
```

Comment:
Why are you adding commented code?

Reply:
I remove the comments from this file and also from ems_block_storages_center.rb